test/e2e/specs/specs-playwright/wp-widgets__spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DataHelper, LoginFlow, SidebarComponent, setupHooks } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const gutenbergUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+describe( DataHelper.createSuiteTitle( 'Widgets' ), function () {
+	let sidebarComponent: SidebarComponent;
+	let page: Page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log in', async function () {
+		const loginFlow = new LoginFlow( page, gutenbergUser );
+		await loginFlow.logIn();
+	} );
+
+	// @todo: Refactor/Abstract these steps into a WidgetsEditor component
+	it( 'Navigate to the Block Widgets Editor', async function () {
+		sidebarComponent = new SidebarComponent( page );
+		await sidebarComponent.gotoMenu( {
+			item: 'Appearance',
+			subitem: 'Widgets',
+			skipWaitForContent: true,
+		} );
+	} );
+
+	it( 'Dismiss the Welcome Gudie Notice', async function () {
+		await page.waitForLoadState( 'networkidle' );
+		const btnSelector = 'button:text("Got it")';
```

Comment:
I'd prefer that this constant either be named
`button`
or
`elementHandle`.

Stylistically, it is preferable to not contract names like in this case.

Reply:
Sounds good 👍🏻